app/views/users/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class='card full-width'>
+  <div class='w-form'>
+  <%= form_for :user, url: users_path do |f| %>
+    <% if @user.errors.any? %>
+      <ul>
+      <% @user.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    <% end %>
+
+    <%= f.label :fullname, 'Full Name', class: 'form-field-label' %>
+    <%= f.text_field :fullname, placeholder: 'Enter name here...', maxlength: '256', class: 'form-field w-input' %>
+   
+    <%= f.label :email, 'Email', class: 'form-field-label' %>
+    <%= f.text_field :email, placeholder: 'Enter email here', class: 'form-field w-input' %> 
+
+    <%= f.label :is_admin, 'Is Admin', class: 'form-field-label' %>
+    <%= f.check_box :is_admin, {}, "true", "false" %> 
+
+    <%= f.label :team_leader_id, 'Team Leader', class: 'form-field-label' %>
+    <%= f.text_field :team_leader_id, placeholder: 'Enter user id', class: 'form-field w-input' %> 
+    
+    <%= f.submit 'Submit', class: 'btn-primary w-button' %>
+    <% end %>
+
+  </div>
```

Comment:
And I think you might have an extra closing div here? Looks like you have one opening tag on line 1 and one on line 2.

Reply:
Yes there was. I removed the extra div from this file

Reply:
I don't think your change got saved, I still see 3 closing div tags here and only two opening div tags.

Reply:
Whoops sorry, I hadn't refreshed the page. Looks good 👍